Resources/views/Collector/migrations.html.twig

```diff
@@ -0,0 +1,192 @@
+{% extends app.request.isXmlHttpRequest ? '@WebProfiler/Profiler/ajax_layout.html.twig' : '@WebProfiler/Profiler/layout.html.twig' %}
+
+{% block toolbar %}
```

Comment:
As this panel mostly brings value when you have unavailable or new migrations, which won't be the case 99% of the cases, maybe it should hide itself from the toolbar for other cases to save space, as done for many core panels ?